dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/resource/core/SearchResource.java

```diff
@@ -82,6 +87,14 @@ public Keywords findAll(
       return EMPTY_RESULT;
     }
 
+    val unknownFields = fields.stream()
+        .filter(f -> FIELD_MAPPING.get(f) == null)
```

Comment:
This type of checking should probably be moved into the service to keep the FIELDS_MAPPING from leaking. A specific exception can then be thrown and then translated to 400 here or in an ExceptionMapper.